unittest/TestShaderGenerators.cpp

```diff
@@ -352,20 +352,18 @@ bool TestShaderGenerators() {
 	init_glslang();
 	LoadD3DCompilerDynamic();
 
-	if (!TestReinterpretShaders()) {
+	if (!TestFragmentShaders()) {
 		return false;
 	}
+	return true;
```

Comment:
Is this return intentional or left from testing?

Reply:
not intentional to leave behind, but inconsequential since this is just test code that doesn't get compiled into the app.